tipg/sql/dbcatalog.sql

```diff
@@ -195,6 +203,29 @@ CREATE OR REPLACE FUNCTION pg_temp.tipg_fproperties(
     SELECT pg_temp.tipg_fproperties(pg_proc) FROM pg_proc WHERE oid=func::regproc;
 $$ LANGUAGE SQL;
 
+CREATE OR REPLACE FUNCTION pg_temp.tipg_get_schemas(include text[] DEFAULT NULL, exclude text[] DEFAULT NULL) RETURNS SETOF oid AS $$
```

Comment:
Create a function that will take use the search_path if the passed in include schemas is null or empty otherwise it will use the include schemas and will exclude those from the exclude schemas as well as any schemas the user does not have permissions for.